hashes/src/lib.rs

```diff
@@ -117,7 +115,7 @@ pub mod serde_macros;
 pub mod cmp;
 pub mod hash160;
 pub mod hmac;
-#[cfg(any(test, feature = "std", feature = "core2"))]
+#[cfg(feature = "bitcoin_io")]
```

Comment:
This has to change to "bitcoin-io", I think that is the cause of all the red in CI.